### ISSUE TITLE
Change Behavior

### ISSUE THREAD

**Issue by rappen** [Innofactor/XrmToolBox.DateTimeBehaviorUtility#5]
Add possibility to change metadata of attributes to change the Behavior.

**Comment by SDPMrem71** [comment]
+1
this feature would really be nice.

**Comment by CharlesBaker-57** [comment]
yes, this would be a great addition...we could use this right now, sigh...

**Comment by veeyeskay** [comment]
Any plans to incorporate this feature anytime soon? One of my customers is looking for this as a critical need.

**Comment by imranakram** [comment]
Hello everyone, this project has been stale for a long time. The original developers haven't been working on it for a long time. I'll try and dig into it and see if we can manage to get this working.

**Comment by fastbone** [comment]
+1
